install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -2555,7 +2555,7 @@ dnsProxy:
   # -- Maximum number of IPs to retain for expired DNS lookups with still-active connections.
   maxDeferredConnectionDeletes: 10000
   # -- The minimum time, in seconds, to use DNS data for toFQDNs policies.
-  minTtl: 3600
+  minTtl: 0
```

Comment:
The comments in `pkg/defaults/defaults.go` mention that the value is a lower bound for the TTL, which is good, but the comment here makes no mention of it. When reading the chart or the README.md or Helm reference generated from it, it may cause confusion as to why this value can be `0`.

Can you please add a comment here to mention that it's a lower bound and `0` means we're using TTLs returned from the upstream DNS, or some similar explanation?

Reply:
ok updated helm value docs:
```
+  # -- The minimum time, in seconds, to use DNS data for toFQDNs policies. If
+  # the upstream DNS server returns a DNS record with a shorter TTL, Cilium
+  # overwrites the TTL with this value. Setting this value to zero means that
+  # Cilium will honor the TTLs returned by the upstream DNS server.
```

Reply:
Naturally Amazing.